charts/ingress-nginx/README.md

```diff
@@ -4,280 +4,108 @@
 
 To use, add the `kubernetes.io/ingress.class: nginx` annotation to your Ingress resources.
 
-## TL;DR;
+This chart bootstraps an ingress-nginx deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.6+
+
+## Get Repo Info
 
 ```console
 $ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-$ helm install my-release ingress-nginx/ingress-nginx
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$ helm repo update
 ```
 
-## Introduction
+## Install Chart
 
-This chart bootstraps an ingress-nginx deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+```console
+# Helm 3
+$ helm install [RELEASE_NAME] ingress-nginx/ingress-nginx
 
-## Prerequisites
+# Helm 2
+$ helm install --name [RELEASE_NAME] ingress-nginx/ingress-nginx
+```
+
+The command deploys ingress-nginx on the Kubernetes cluster in the default configuration.
 
-  - Kubernetes 1.6+
+_See [configuration](#configuration) below._
 
-## Installing the Chart
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
 
-To install the chart with the release name `my-release`:
+## Uninstall Chart
 
 ```console
-$ helm install --name my-release ingress-nginx/ingress-nginx
-```
+# Helm 3
+$ helm uninstall [RELEASE_NAME]
 
-The command deploys ingress-nginx on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+# Helm 2
+# helm delete --purge [RELEASE_NAME]
+```
 
-> **Tip**: List all releases using `helm list`
+This removes all the Kubernetes components associated with the chart and deletes the release.
 
-## Uninstalling the Chart
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
 
-To uninstall/delete the `my-release` deployment:
+## Upgrading Chart
 
 ```console
-$ helm delete my-release
+# Helm 3 or 2
+$ helm upgrade [RELEASE_NAME] [CHART] --install
 ```
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
+_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
-## Configuration
+### Upgrading With Zero Downtime in Production
 
-The following table lists the configurable parameters of the ingress-nginx chart and their default values.
```

Comment:
just remove this section without a replacement?

Reply:
Yeah, why was this removed?

Reply:
Hey good question – I should have given more context, have been doing this a lot lately trying to prep the move from stable for charts that have been in discussion for a while.

These chart README templates have been copied and re-copied across charts for years, and now that we're moving repos and counting down the days to Helm 2's end of support, it seems like a good time to simplify things.

For this issue, I needed to add an Upgrading section, for the remaining item in #5161 before we can mark the `stable/nginx-ingress` chart deprecated and point users here. I took the opportunity to make a few other section cleanup changes.  Major differences are:
- Added Helm 2 and Helm 3 commands, with links to Helm 3 command reference links (from there, users can select Helm 2 from the version dropdown if they wish)
- Removed the options table that's duplicated from (often out of sync with) `values.yaml` in favor of `helm inspect` (Helm 2) and `helm show values` (Helm 3). The `values.yaml` file is well organized and commented, giving users a much better (and never out of sync) reference to what they can configure.

The rest of the changes I made were capitalization, removing some other duplicated instructions, linking to the better docs now at helm.sh/docs (they were not always this complete in the past when these README templates were initially written), nesting sub-headers (the configuration-related H2s now H3s underneath the "configure" H2, etc).

Related discussion at https://github.com/scottrigby/prometheus-helm-charts/pull/14, where we're moving all the prometheus-related charts.

Personally I think it's cleaner this way, but lmk what you think 🙂

Reply:
This is great. I had no idea about the helm show values. I see now that is commented down later.

Reply:
> Removed the options table that's duplicated from (often out of sync with) values.yaml in favor of helm inspect (Helm 2) and helm show values (Helm 3).

Makes sense. +1 from me